diffrax/brownian/tree.py

```diff
@@ -34,12 +42,58 @@ class _State(eqx.Module):
     w_t: Scalar
     w_u: Scalar
     key: "jax.random.PRNGKey"
+    J_s: Optional[Scalar] = None
+    J_t: Optional[Scalar] = None
+    J_u: Optional[Scalar] = None
+
+
+def _wj_to_wh_diff(x0: LevyVal, x1: LevyVal) -> LevyVal:
+    r"""Computes $(W_{s,u}, H_{s,u})$ from $(W_s, J_s)$ and
+    $(W_u, J_u)$, where $J_u = \int_0^u W_t dt$
+
+    **Arguments:**
+
+        - `x0`: LevyVal$(W_s, J_s)$
+        - `x1`: LevyVal$(W_u, J_u)$
```

Comment:
Nits: dedent these two lines. Also, I'm not sure what's going on with the partial use of LaTeX. I think I'd suggest just writing it as code, `LevyVal(W_s, J_s)`, if it's annoying to express in LaTeX.